package/MDAnalysis/analysis/align.py

```diff
@@ -380,6 +382,177 @@ def alignto(mobile, reference, select="all", mass_weighted=False,
     return old_rmsd, new_rmsd
 
 
+class AlignTraj(AnalysisBase):
+    def __init__(self, mobile, reference, select='all', filename=None, prefix='rmsfit_',
+                    mass_weighted=False, tol_mass=0.1, strict=False, force=True, quiet=False,
+                    start=None, stop=None, step = None, **kwargs):
+        """RMS-fit trajectory to a reference structure using a selection.
+
+        Both reference *ref* and trajectory *traj* must be
+        :class:`MDAnalysis.Universe` instances. If they contain a
+        trajectory then it is used. The output file format is determined
+        by the file extension of *filename*. One can also use the same
+        universe if one wants to fit to the current frame.
+
+        :Arguments:
```

Comment:
On it, I have 4 hours blocked tonight to do finish everything up. Some other work I need to get done right now.